students/Kristy_Martini/lesson09/mailroom.py

```diff
@@ -0,0 +1,149 @@
+import operator
+import os
+
+class Report:
+    def __init__(self):
+        """ Report class holds the list of all donors and information associated with them."""
+        self.donors = dict()
+        Kristy = Donor("Kristy Martini", 0, 3, 3000)
+        Mikey = Donor("Mike Martini", 0, 7, 3424834)
+        Cathy = Donor("Cathy Martini", 0, 7, 63833)
+        Bill = Donor("Bill Martini", 0, 10, 60000)
+        Nick = Donor("Nick Martini", 0, 50, 47484949)
+
+        initial_donors = [Kristy, Mikey, Nick, Bill, Cathy]
+        [self.add_donor(donor) for donor in initial_donors]
```

Comment:
this is an anti-pattern, your comprehension returns a list which should be stored into something.

in your case you should just use a regular for loop.